src/main/java/com/venafi/vcert/sdk/utils/VCertUtils.java

```diff
@@ -0,0 +1,82 @@
+package com.venafi.vcert.sdk.utils;
+
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+
+import com.venafi.vcert.sdk.certificate.CertificateRequest;
+import com.venafi.vcert.sdk.connectors.tpp.AbstractTppConnector.CertificateRequestsPayload;
+import com.venafi.vcert.sdk.connectors.tpp.AbstractTppConnector.NameValuePair;
+
+public class VCertUtils {
+
+	public static void addExpirationDateAttribute( CertificateRequest request, CertificateRequestsPayload payload ) {
+
+		if ( request.validityHours() > 0 ) {
+
+			Instant now = Instant.now();
+			LocalDateTime utcTime = LocalDateTime.ofInstant(now, ZoneOffset.UTC);
+
+			int validityDays = request.validityHours() / 24;
+
+			if ( request.validityHours() % 24 > 0 ) {
```

Comment:
Can you put a little comment to explain why the validityDays is plus to 1 if the remainder of the division by 24 is more than 0?

Reply:
fixed on the most recent commit